be/src/olap/storage_engine.h

```diff
@@ -53,6 +52,7 @@ namespace doris {
 class Tablet;
 class DataDir;
 class EngineTask;
+class SegmentGroup;
```

Comment:
why add SegmentGroup here? I think segment group should not be used here.

Reply:
because `_gc_files` use this

Reply:
_gc_files is useless, delete it and SegmentGroup from storage engine